rsmtool/preprocess.py

```diff
@@ -177,99 +263,230 @@ def remove_outliers(data, mean=None, sd=None, sd_multiplier=4):
 
 
 def apply_inverse_transform(name, data, sd_multiplier=4):
-
     """
     Apply the inverse transform to `data`.
+
+    Parameters
+    ----------
+    name : str
+        Name of the feature to transform.
+    data : numpy array
+        Numpy array containing the feature values.
+    sd_multiplier : int, optional
```

Comment:
"Use this std. dev. multiplier to compute the ceiling and floor for outlier removal and check that these are not equal to zero"